FastBank.ConsoleApplication/Program.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using FastBank.Domain;
 using FastBank.Services;
+using System.Text;
 
 namespace FastBank
 {
     public class Program
     {
         static void Main(string[] args)
         {
+            Console.OutputEncoding = Encoding.UTF8;
```

Comment:
Why?

Reply:
![image](https://github.com/bulgarchik/FastBank/assets/41870483/2662618a-c41a-4510-85d6-fab55ef86afa)
We have Cyrillic symbols